requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyyaml
```

Comment:
Can you pin these versions to whatever the latest release/whichever worked for you?